src/quest/baseQuest.js

```diff
@@ -47,4 +49,17 @@ export class BaseQuest {
 
     }
 
+    static async getDirectory(language) {
+        var downPath = "";
+        if (language === "solidity") {
+            downPath = `campaigns/directory.json`;
+        } else {
+            downPath = `campaigns/${language}-directory.json`;
+        }
+        const qdown = new QuestDownloader();
+        await qdown.downloadFile("Nodeguardians", "ng-questplay", downPath);
+
+        return path.join(process.cwd(), downPath);
+    }
```

Comment:
Oops, I think I wasn't clear in my explanation here. The idea is to drop all instances of `{LANG}-directory.json` files in Questplay. Instead, users will download the directory from:
1. [Solidity directory](https://github.com/Nodeguardians/ng-quests-public/blob/main/campaigns/directory.json)
2. [Cairo directory](https://github.com/Nodeguardians/ng-cairo-quests-public/blob/main/campaigns/directory.json)
3. [Noir directory](https://github.com/Nodeguardians/ng-noir-quests-public/blob/main/campaigns/noir-directory.json)
4. [Huff directory](https://github.com/Nodeguardians/ng-huff-quests-public/blob/main/campaigns/huff-directory.json)

Reply:
But yeah, this also means:
* Deleting the old directory files in this repository.
* Not writing anything onto the repository (e.g., keeping directory.json as temporary data, not written to disk)

Reply:
To achieve the latter by adding something like this in `QuestDownloader.js`
```
getDirectory(..., options)
const file = await this._octokit.repos.getContent({
  owner,
  repo,
  ref: options.sha,
  path: zipFilePath,
});

const decodedContent = Buffer.from(
  file.data.content, 
  file.data.encoding
);

return JSON.parse(decodedContent);
```
I didn't test this code block, but it should be something like that.

Reply:
Done, but the huff public repo doesn't exist yet. Since I assume this PR will only get merged in once the huff quests are published, I add the huff stuff in preparation for when it'll be live. Let me know if that's good.